components/eam/bld/build-namelist

```diff
@@ -2875,10 +2875,11 @@ if ($chem eq 'linoz_mam4_resus_mom' || $chem eq 'linoz_mam4_resus_mom_soag') {
 
     my %species;
 
-    # Surface emission datasets:
+    # Surface emission datasets (AH -- added E90 here):
```

Comment:
I would remove the signatures, we can always find diffs using git diff, and attribute changes via git blame.